quiz/backend/stats/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+
+
+class TimestampModel(models.Model):
```

Comment:
I wonder if this model should be placed in here or in some kind of utils for all apps.

Reply:
Sure, could also place this model in `helpers`

Reply:
So I was thinking about it for now I would like to leave time stamp model in this file, because I'm not sure if models from any other app, are going to use it